### ISSUE TITLE
[FEATURE] Port blur from lawnchair 2

### ISSUE THREAD

**Issue by Jonas2389** [Deletescape-Media/Lawnchair#4852]
### Describe the feature

Please port option to add a blur effect to the app drawer background. This would not only provide a sleek and modern look but also help in focusing on the app icons, making the drawer visually appealing while maintaining a clean interface when opacity is low.

### Why would it be useful to add?

For aesthetics

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

![Screenshot_20240923_133437_Lawnchair](https://github.com/user-attachments/assets/eb4cc734-e6e9-4732-bf55-2a23c8aa7a82)
![Screenshot_20240923_133426_Lawnchair](https://github.com/user-attachments/assets/fe2ef086-f696-45c8-b7d7-1652f5565f35)

**Comment by sarojpanda44** [comment]
Isn't it same as Background Opacity?
![Screenshot_20240925-093715](https://github.com/user-attachments/assets/e18b39a0-d50d-4670-95c4-1a586fc99cd8)

**Comment by sarojpanda44** [comment]
![Screenshot_20240925-093850](https://github.com/user-attachments/assets/646e44c7-1827-4bf6-8ce9-aa250594646c)

**Comment by 2004seraph** [comment]
> Isn't it same as Background Opacity?

The blur kernel seems to be constant with background opacity, it feels like a visual enhancement added on top if the transparency. I believe OP wants a more aggressive blur kernel as an option, which would look more similar to the acrylic look on Windows. I would also like this.

**Comment by ManuelBoe14** [comment]
It would be really nice to have the blur option back. It looks aesthetically way more pleasing while allowing for better readability when setting the opacity level to a very low level. There is an option in the experimental settings to enable blur but it's not the same because it blurs the whole background. It doesn't behave the same as the old v2 one did.

**Comment by gpolydoros** [comment]
> > Isn't it same as Background Opacity?
> 
> The blur kernel seems to be constant with background opacity, it feels like a visual enhancement added on top if the transparency. I believe OP wants a more aggressive blur kernel as an option, which would look more similar to the acrylic look on Windows. I would also like this.

Indeed, it differs substantially from the acrylic-like look that SystemUI and the new Pixel Launcher have with the new Android 16 QPR1 beta. It would be nice to have the option to customize the blur intensity to match that new look with Lawnchair.

**Comment by sppidy** [comment]
@Jonas2389 the below is the background opacity / blur in latest nightly build. Is this what you are expecting ? If yes we could close the issue

<img width="1220" height="2712" alt="Image" src="https://github.com/user-attachments/assets/34bae469-01aa-4f0a-98ee-760b71b98c99" />

**Comment by sppidy** [comment]
@SuperDragonXD 
This issue can be closed within the next 24 hrs if there is no response. Also there should be support for window level blur support from the rom to get the acrylic blur working. 

I will attach 2 screenshots which describes what I am saying

![Image](https://github.com/user-attachments/assets/3fd9ab54-32ec-4633-90ab-1d6d52384cb5)

![Image](https://github.com/user-attachments/assets/7f83767a-5176-4078-9d06-66e4e25154c5)

**Comment by SuperDragonXD** [comment]
If I remember correctly, window blurring is only supported on Android 12+ devices (with that option enabled). So we need to also have a fallback option for older devices.
